charts/argocd/values.yaml

```
@@ -20,15 +20,17 @@ argo-cd:
                 Argo CD has not reported any applications data for the past 15 minutes which
                 means that it must be down or not functioning properly.  This needs to be
                 resolved for this cloud to continue to maintain state.
-          - alert: ArgoAppNotSynced
-            expr: |
-              argocd_app_info{sync_status!="Synced"} == 1
-            for: 12h
+          - alert: ArgoCDSync
+            expr: label_replace(argocd_app_info{sync_status!="Synced"}, "namespace", "$1", "dest_namespace", "(.*)") > 0
+            for: 60m
             labels:
               severity: warning
             annotations:
-              summary: "[{{`{{$labels.name}}`}}] Application not synchronized"
-              description: >
-                The application [{{`{{$labels.name}}`}} has not been synchronized for over
-                12 hours which means that the state of this cloud has drifted away from the
-                state inside Git.
+              summary: The ArgoCD application is not syncronized.
```

I would prefer that the summary still includes the label of the application that is not synchronized. Maybe we leave the summary like before and keep the description more verbose like you proposed?

Also, have you tested this? If I recall correctly, I remember @fsismondi having a few issues with this alert  a while back, because it was not rendering correctly with the label not being parsed in the summary/description.

```suggestion
              summary: "[{{`{{$labels.name}}`}}] Application not synchronized"
```

---

I dont recall anymore if we solved this name extrapolation issue or not. This alert went under the radar as we stopped getting notified about it while BGIS refactored the alerting solution. Maybe @omohammed95 you can check this while you are it?

---

Removing extra brackets fixes the label issue.